NuGet.config

```diff
@@ -5,6 +5,7 @@
   </fallbackPackageFolders>
   <packageSources>
     <clear />
+     <add key="darc-pub-dotnet-maintenance-packages-f9ed1720" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/darc-pub-dotnet-maintenance-packages-f9ed1720/nuget/v3/index.json" />
```

Comment:
This should be moved to the other darc feed.  It would be helpful to add a comment like the other feed as well.

Reply:
Sorry, can you elaborate?

Reply:
Sorry my wording was poor.  Please move it next to the other existing darc feed - https://github.com/dotnet/source-build-reference-packages/blob/main/NuGet.config#L24

Reply:
Ah! Sure, will do.